scripts/metrics/ctest_metrics_xml_to_csv.py

```diff
@@ -47,9 +46,8 @@ def main():
 
     # Create csv file
     full_path = os.path.join(args.output_directory, args.branch)
-    if args.weeks:
-        week = datetime.datetime.now().isocalendar().week
-        full_path = os.path.join(full_path, f"Week{week:02d}")
+    week = datetime.datetime.now().isocalendar().week
```

Comment:
Today and one year from now appear to map into the same folder, which can create a reporting issue. I think that week needs to be preceded by year, though it may be more appropriate to instead have folders for year/month/day and let any reporting scripts ingest whichever days they require.

Reply:
I was planning on having an archive folder for other years, but perhaps that may be overcomplicating it and years can just be included in the folder path. Preference?

Reply:
That archive pattern creates an odd boundary condition when one year transitions to the next. I think archiving should be set by a data retention policy, not by the folder structure.

I think `branch/year/month/day` existing as folders makes sense, with timestamp and for UUID the filename.